### ISSUE TITLE
BCIERS Dashboard

### ISSUE THREAD

**Issue by shon-button** [bcgov/cas-registration#1653]
**Description:**

For the IA (copy/text, links, navigation, etc.), please refer to the frames in Figma below:
- [BCIERS External - Dashboard page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7692-3728&t=pmKjK1RmQeLYOb04-4)
- [BCIERS Internal - Dashboard page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7781-6471&t=pmKjK1RmQeLYOb04-4)

For the styling (font, colour, margin, padding, etc.), please refer to [this frame on the BCIERS Design Guidelines page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=11103-107&t=pmKjK1RmQeLYOb04-4) in Figma.

#### Acceptance Criteria:

Given I'm an Operator Admin
When I log into BCIERS
Then I see the BCIERS dashboard with all 4 tiles as in the [design](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7692-3728&t=pmKjK1RmQeLYOb04-4)
When I click on the sub links of each tile
Then I'm navigated to the proper page as in the [prototype](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?page-id=7692%3A2&type=design&node-id=7692-3728&viewport=153%2C235%2C0.04&t=9CHcuF7tZEcxkUP7-1&scaling=scale-down&starting-point-node-id=7692%3A3728&show-proto-sidebar=1)

Given I'm an IRC Admin
When I log into BCIERS
Then I see the BCIERS dashboard with all 5 tiles as in the [design](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7781-6471&t=pmKjK1RmQeLYOb04-4)
When I click on the sub links of each tile
Then I'm navigated to the proper page as in the [prototype](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?page-id=7781%3A6398&type=design&node-id=7781-6471&viewport=74%2C199%2C0.46&t=xuYjeIKfjwjoCJNb-1&scaling=scale-down&starting-point-node-id=7781%3A6471)

#### Development Checklist:

- [ ] When creating placeholder pages, carefully consider the folder structure and file naming. The structure determines our routes and breadcrumbs (both can be tweaked but it's better if the structure is as organized as possible)
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- ~~Blocked by #1621 , #1577~~
- Blocking

**Comment by suhafa** [comment]
Is this ticket just for capturing the links? Or is it for the styling? @nanyangpro and I had discussed that the styling of this is not the way we are are going about this. This is the [lastest external user common dashboard design](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7789%3A884&t=i99A8cQ2Z2W6IGzP-1) @shon-button @hannavovk

**Comment by hannavovk** [comment]
@nanyangpro, can you update and refine this ticket? e.g. update the screenshots, include AC and current links, etc. (cc: @suhafa, @patrickisaac )

**Comment by nanyangpro** [comment]
Hi @shon-button (cc: @hannavovk, @suhafa) - I have updated the links to the proper frames in Figma to show the **page details** and **styling details** separately. Happy to hop on a call if you have questions or need help fine-tuning the AC together.

**Comment by suhafa** [comment]
**Note for devs** who will be working on this, the Internal dashboard design linked here will follow the same design style as the External dashboard as mentioned by Nan. For the 'Clean Growth User Access Management' tile in the Internal side, you can use the [first icon that appears when you click this link](https://feathericons.com/?query=users) @shon-button

**Comment by patriciarussellCAS** [comment]
@hannavovk I tried reviewing this but couldn't get the link to work using IDIR log-in. Perhaps we could sync to discuss PO approval of this card?

**Comment by shon-button** [comment]
@patriciarussellCAS

Reachable at:
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/

**Comment by patriciarussellCAS** [comment]
Unable to confirm what this looks like for internal users, but external users looks good. Will close this.
